src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/Caseload.kt

```diff
@@ -1,29 +1,28 @@
 package uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa
 
+import jakarta.persistence.Column
+import jakarta.persistence.Convert
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.OneToMany
+import jakarta.persistence.Table
 import org.hibernate.Hibernate
-import org.hibernate.annotations.BatchSize
-import org.hibernate.annotations.Type
-import javax.persistence.Column
-import javax.persistence.Entity
-import javax.persistence.FetchType
-import javax.persistence.Id
-import javax.persistence.JoinColumn
-import javax.persistence.OneToMany
-import javax.persistence.Table
+import org.hibernate.type.YesNoConverter
 
 const val DPS_CASELOAD = "NWEB"
 const val CENTRAL_ADMIN_CASELOAD = "CADM_I"
 
 @Entity
 @Table(name = "CASELOADS")
-@BatchSize(size = 100)
 data class Caseload(
```

Comment:
Is this no longer needed?

Reply:
@SimonMitchellMOJ - please reply to this as I'm not sure

Reply:
I think it might still be needed was added as part of HAAR-470

Reply:
Re-instated